services/src/main/scala/cromwell/pipeline/service/UserService.scala

```diff
@@ -62,22 +62,23 @@ object UserService {
         request: PasswordUpdateRequest,
         salt: String = Random.nextLong().toHexString
       ): Future[Int] =
-        if (request.newPassword == request.repeatPassword) {
-          userRepository.getUserById(userId).flatMap {
-            case Some(user) =>
-              user match {
-                case user
-                    if user.passwordHash == calculatePasswordHash(request.currentPassword.unwrap, user.passwordSalt) => {
-                  val passwordSalt = salt
-                  val passwordHash = calculatePasswordHash(request.newPassword.unwrap, passwordSalt)
-                  userRepository.updatePassword(user.copy(passwordSalt = passwordSalt, passwordHash = passwordHash))
-                }
-                case _ => Future.failed(new RuntimeException("user password differs from entered"))
-              }
-            case None => Future.failed(new RuntimeException("user with this id doesn't exist"))
-          }
-        } else Future.failed(new RuntimeException("new password incorrectly duplicated"))
-
+        request.newPassword match {
+          case request.repeatPassword =>
+            userRepository.getUserById(userId).flatMap {
+              case Some(userWithCredentials)
```

Comment:
I think. we don't need to write this value as `userWithCredentials`, I think that` user` should be enough.

Reply:
I did not change it because it better reflects reality.

Reply:
Ok. As you wish.